client/src/utils/fetch.js

```diff
@@ -19,7 +19,7 @@ const baseRequest = {
   credentials: 'include',
 };
 
-async function _getAPI(uri) {
+const _getAPI = async uri => {
```

Comment:
suggestion: #readability while uri is correct, reading uri vs. url can be confusing and error prone. Might we rename "uri" to resource and improve clarity?

This suggestion seems out of scope.

Reply:
https://github.com/GoogleCloudPlatform/avocano/issues/339